change/@azure-msal-node-e1216fab-db96-4b5e-b319-17f3bcffa13b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "ClientAssertion.parseCertificate - allow newlines in cert",
```

```suggestion
  "comment": "ClientAssertion.parseCertificate - allow newlines in cert (#2721).",
```

This adds in the PR number to the changelog as well.